pubspec.yaml

```diff
@@ -98,6 +100,8 @@ dependencies:
   iconify_flutter: ^0.0.5
   # Convenient code generator for get_it
   injectable: ^2.3.0
+  injectable_generator: ^2.1.3 # Version 2
```

Comment:
`injectable_generator` already exists in `dev_dependencies`.
Please update the version there

Reply:
You removed it entirely instead of updating it.
Please update the existing one